spec/swagger/swagger_doc_v1.yaml

```diff
@@ -448,6 +456,59 @@
             application/vnd.api+json:
               schema:
                 "$ref": "../v1/post_documents_responses.yaml#/422"
+  "/framework_responses/{framework_response_id}":
+    patch:
+      summary: Updates a framework response's value
+      tags:
+      - FrameworkResponse
```

Comment:
Is this what maps to the heading? If so, could it be a plural to keep consistency with the other headings. And if possible, could it contain a space - `Framework responses` :D?